components/gantry/multiaxis/multiaxis_test.go

```diff
@@ -37,8 +37,8 @@ func createFakeOneaAxis(length float64, positions []float64) *inject.Gantry {
 	fakesingleaxis.CloseFunc = func(ctx context.Context) error {
 		return nil
 	}
-	fakesingleaxis.ModelFrameFunc = func() referenceframe.Model {
-		return nil
+	fakesingleaxis.KinematicsFunc = func(ctx context.Context) (referenceframe.Model, error) {
+		return referenceframe.NewSimpleModel(""), nil
```

Comment:
This is impossible isn't it given all gantry clients return unimplemented error?

Reply:
I don't understand why this would be impossible.  SingleAxis gantrys do implement this function